src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,94 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int DEFAULT_INCREASE_CAPACITY = 2;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    private int size;
+    private float threshold;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+    }
+
+    private static class Node<K, V> {
+        private int hash;
```

Comment:
as i can see we don't use this variable. In this case let's remove it

Reply:
removed